src/pages/document/[id].tsx

```diff
@@ -167,7 +167,7 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({
   };
 
   return (
-    <Layout title={`${page.title}`} description={getFamilyMetaDescription(page.summary, geographyNames?.join(", "), page.category)}>
+    <Layout title={`${page.title}`} description={getFamilyMetaDescription(page.summary, geographyNames?.join(", "), page.category)} appName={theme}>
```

Comment:
I know there might be past reasoning for this, but the swapping between `app` and `theme` gets a bit muddled in my head. 

My understanding of this is that we have multiple apps. Each app might have different styling. Is that the `theme`?

Reply:
Confusion is fair. I think as an org we have some inconsistency referring to them as "custom apps" and "themes".

Reply:
But yes your comment is correct. My preference for an explanation of the code is:

This repo is a theme _factory_, that can build apps based upon a selected theme.